common/lib/capa/capa/tests/test_inputtypes.py

```diff
@@ -781,25 +783,17 @@ def test_get_html(self):
             """).replace('\n', '')
 
         output = self.the_input.get_html()
+        output_string = etree.tostring(output).decode('utf-8')
         if six.PY2:
-            expected_string = """
-            <div class='matlabResponse'><div id='mwAudioPlaceHolder'>
-            <audio src='data:audio/wav;base64=' autobuffer="" controls="" autoplay="">Audio is not supported on this browser.</audio>
-            <div>Right click <a href="https://endpoint.mss-mathworks.com/media/filename.wav">here</a> and click \"Save As\" to download the file</div></div>
-            <div style='white-space:pre' class='commandWindowOutput'/><ul/></div>
-            """
+            self.assertRegexpMatches(output_string, "div.*mwAudioPlaceHolder")
+            self.assertRegexpMatches(output_string, "audio.*Audio is not supported.*audio")
+            self.assertRegexpMatches(output_string, "div.*Right click.*href.*media.*here.*click.*download.*div")
+            self.assertRegexpMatches(output_string, "div .*style=.*ul.*div")
         else:
-            expected_string = """
-            <div class="matlabResponse"><div id="mwAudioPlaceHolder">
-            <audio autobuffer="" controls="" autoplay="" src="data:audio/wav;base64=">Audio is not supported on this browser.</audio>
-            <div>Right click <a href="https://endpoint.mss-mathworks.com/media/filename.wav">here</a> and click \"Save As\" to download the file</div></div>
-            <div style="white-space:pre" class="commandWindowOutput"/><ul/></div>
-            """
-        self.assertEqual(
-            etree.tostring(output).decode('utf-8'),
-            textwrap.dedent(expected_string).replace('\n', '').replace('\'', '\"')
-        )
-
+            self.assertRegex(output_string, "div.*mwAudioPlaceHolder")
+            self.assertRegex(output_string, "audio.*Audio is not supported.*audio")
+            self.assertRegex(output_string, "div.*Right click.*href.*media.*here.*click.*download.*div")
+            self.assertRegex(output_string, "div .*style=.*ul.*div")
```

Comment:
And [six.assertRegex](https://six.readthedocs.io/#six.assertRegex) might be a better choice here.  Or alternatively, parse the HTML and make assertions about its structure.